lib/src/services/music.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:nyxx/nyxx.dart';
+import 'package:nyxx_commands/nyxx_commands.dart';
+import 'package:nyxx_lavalink/nyxx_lavalink.dart';
+import 'package:running_on_dart/running_on_dart.dart';
+import 'package:running_on_dart/src/util.dart';
+
+/// The address of the lavalink running server to connect to.
+String serverAddress = getEnv('LAVALINK_ADDRESS', 'localhost');
```

Comment:
Would it be possible to include lavalink as a service in the docker-compose and use that? If you're unsure how to do it just ping me and I'll set it up.

Reply:
Well, i've never really used docker, so i'm not sure how to do it